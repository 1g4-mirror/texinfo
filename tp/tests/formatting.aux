\relax 
\providecommand\zref@newlabel[2]{}
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\zref@newlabel{mdf@pagelabel-1}{\default{}\page{xi}\abspage{11}\mdf@pagevalue{11}}
\pgfsyspdfmark {pgfid1}{7030046}{27700461}
\zref@newlabel{mdf@pagelabel-2}{\default{}\page{xxiv}\abspage{24}\mdf@pagevalue{24}}
\pgfsyspdfmark {pgfid2}{10568990}{27700461}
\newlabel{anchor:Top}{{}{2}{\contentsname \@mkboth {\MakeUppercase \contentsname }{\MakeUppercase \contentsname }}{chapter*.11}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}chapter}{2}{chapter.1}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{anchor:chapter}{{1}{2}{chapter}{chapter.1}{}}
\zref@newlabel{mdf@pagelabel-3}{\default{1}\page{12}\abspage{39}\mdf@pagevalue{12}}
\pgfsyspdfmark {pgfid3}{10568990}{27700461}
\zref@newlabel{mdf@pagelabel-4}{\default{1}\page{25}\abspage{52}\mdf@pagevalue{25}}
\pgfsyspdfmark {pgfid4}{7030046}{27700461}
\zref@newlabel{mdf@pagelabel-5}{\default{1}\page{42}\abspage{69}\mdf@pagevalue{42}}
\pgfsyspdfmark {pgfid5}{10568990}{18069939}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}A section}{49}{section.1.1}\protected@file@percent }
\newlabel{anchor:s_002d_002dect_002cion}{{1.1}{49}{A section}{section.1.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1.1}subsection}{49}{subsection.1.1.1}\protected@file@percent }
\newlabel{anchor:subsection}{{1.1.1}{49}{subsection}{subsection.1.1.1}{}}
\newlabel{anchor:anchor}{{1.1.1}{49}{subsection}{subsection.1.1.1}{}}
\@writefile{toc}{\contentsline {subsubsection}{subsubsection ``simple-double--}{49}{section*.30}\protected@file@percent }
\newlabel{anchor:subsubsection-_0060_0060simple_002ddouble_002d_002d}{{1.1.1}{49}{subsubsection ``simple-double--}{section*.30}{}}
\@writefile{toc}{\contentsline {subsubsection}{three---four----''}{49}{section*.31}\protected@file@percent }
\newlabel{anchor:subsubsection-three_002d_002d_002dfour_002d_002d_002d_002d_0027_0027}{{1.1.1}{49}{three---four----''}{section*.31}{}}
\newlabel{anchor:chapter2}{{1.1.1}{49}{\centering chapter 2}{chapter*.32}{}}
\gdef \@abspage@last{77}
