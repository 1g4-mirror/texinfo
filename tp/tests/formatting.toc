\contentsline {chapter}{\numberline {1}chapter}{2}{chapter.1}%
\contentsline {section}{\numberline {1.1}A section}{49}{section.1.1}%
\contentsline {subsection}{\numberline {1.1.1}subsection}{49}{subsection.1.1.1}%
\contentsline {subsubsection}{subsubsection ``simple-double--}{49}{section*.30}%
\contentsline {subsubsection}{three---four----''}{49}{section*.31}%
