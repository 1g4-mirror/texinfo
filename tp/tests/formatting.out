\BOOKMARK [0][-]{chapter.1}{\376\377\000c\000h\000a\000p\000t\000e\000r}{}% 1
\BOOKMARK [1][-]{section.1.1}{\376\377\000A\000\040\000s\000e\000c\000t\000i\000o\000n}{chapter.1}% 2
\BOOKMARK [2][-]{subsection.1.1.1}{\376\377\000s\000u\000b\000s\000e\000c\000t\000i\000o\000n}{section.1.1}% 3
